src/components/CVcomponents/FunctionDescription.js

```diff
@@ -44,9 +49,12 @@ return (
       }
 
       {isDescClicked ?
-        <textarea defaultValue={description} className="description" id="description" autoFocus onBlur={changeDescription}/>
+        <div>
+        <textarea defaultValue={description} className="description" id="description" autoFocus onBlur={changeDescription} onChange={wordCount} maxLength="1000"/>
+        <p id="counter">{isWordCount}/1000 karakters over</p>
```

Comment:
Je wordCount component zit over de tekst die je aan het editen bent. Check ff je styling hiervoor.
![image](https://user-images.githubusercontent.com/17408017/67012731-31d05700-f0f2-11e9-877d-6180558f1c94.png)

Reply:
Because of Chrome, I fixed it, and also added auto resizing when typing